Lib/test/test_asyncio/test_sslproto.py

```diff
@@ -550,29 +550,30 @@ def connection_lost(self, exc):
                 server_side=True,
                 ssl_handshake_timeout=self.TIMEOUT)
 
-            await on_eof
+            proto.replace_transport(new_tr)
+
             await on_con_lost
             self.assertEqual(proto.data, HELLO_MSG)
             new_tr.close()
 
         async def run_main():
             on_con = self.loop.create_future()
-            on_eof = self.loop.create_future()
             on_con_lost = self.loop.create_future()
-            proto = ServerProto(on_con, on_eof, on_con_lost)
+            proto = ServerProto(on_con, on_con_lost)
 
             server = await self.loop.create_server(
                 lambda: proto, '127.0.0.1', 0)
             addr = server.sockets[0].getsockname()
 
             with self.tcp_client(lambda sock: client(sock, addr),
-                                 timeout=self.TIMEOUT):
+                                 timeout=self.TIMEOUT) as threaded_client:
```

Comment:
the new "as" is not used

Reply:
Sorry.
This is an artifact of my tries to fix this test in several ways

Reply:
removed